run_uts.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -x
+set -e
+date
+pwd
+git status
+
+./create_binary.sh
+docker stop pyinstaller docopt
+docker rm pyinstaller docopt
+docker run -rm -v `pwd`/:/code calico-build bash -c '/tmp/etcd & nosetests -c nose.cfg'
+
+# BE CAREFUL
```

Comment:
I agree with Tom that this comment is far too cryptic to be useful.  Why is it cryptic at all?  If there is something important to say then say it explicitly rather than leaving as an exercise for the reader.